components/Order/index.rst

```diff
@@ -4,10 +4,12 @@ Order
 E-Commerce PHP library for creating and managing sales orders.
 
 .. toctree::
-   :numbered:
+   :maxdepth: 2
 
    installation
-   order_and_items
-   adjustments
+   models
+   interfaces
    order_states
-   summary
+   order_events
+   order_transitions
+   basic_usage
```

Comment:
For me more logical is to start with introduce structure of component and then show how it can be used :)